test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectMiddlewareTests.cs

```diff
@@ -151,20 +150,17 @@ public class OpenIdConnectMiddlewareTests
 
         private static void SetProtocolMessageOptions(OpenIdConnectOptions options)
         {
-            var mockOpenIdConnectMessage = new Mock<OpenIdConnectMessage>();
-            mockOpenIdConnectMessage.Setup(m => m.CreateAuthenticationRequestUrl()).Returns(ExpectedAuthorizeRequest);
-            mockOpenIdConnectMessage.Setup(m => m.CreateLogoutRequestUrl()).Returns(ExpectedLogoutRequest);
             options.AutomaticAuthentication = true;
             options.Events = new OpenIdConnectEvents()
             {
                 OnRedirectToAuthenticationEndpoint = (context) =>
                 {
-                    context.ProtocolMessage = mockOpenIdConnectMessage.Object;
+                    context.ProtocolMessage = new OpenIdConnectMessage(ExpectedAuthorizeRequest);
```

Comment:
It is most certainly different but seems to get the same result.  I suspect that I'm abusing some internal behavior of OpenIdConnectMessage, but without a mocking framework to directly replace the return values from those methods I was at a loss for what to do.  This seemed to make the endpoint behave the same way in the tests so I went with it.

I heard that a supported version of moq was coming, but is there an alternative in the meantime?